### ISSUE TITLE
Proper releases

### ISSUE THREAD

**Issue by jacksongoode** [uowuo/abaddon#217]
Maybe it's not time for the project but having a CI/CD & releases for each platform seem like a nice goal!

**Comment by TheMorc** [comment]
<img width="544" alt="image" src="https://github.com/uowuo/abaddon/assets/13377926/34965a47-83c4-47e4-9249-4e3ab50852b0">

well, there are CI builds already (when checking in the Actions tab) but my guess is that only those for Windows are properly packaged right now. I actually tried remaking and fixing the macOS builds yesterday but I failed and left it off for now. (it's just some casual problems with libraries as they need to be packaged and carried in the .app and I don't have experience with static linking to an executable instead of relying on dylibs and stuff)

**Comment by ouwou** [comment]
yeah windows is the only thing thats packaged rn cuz its what im familiar with. im open to having packaging done for macos + linux but i probably wouldnt be the one to do it since i wouldnt really know what im doing there (appimage? snap? flatpak? something else??)

**Comment by tawnylure** [comment]
> yeah windows is the only thing thats packaged rn cuz its what im familiar with. im open to having packaging done for macos + linux but i probably wouldnt be the one to do it since i wouldnt really know what im doing there (appimage? snap? flatpak? something else??)

You'd probably want to package in these
.deb - sense debian / ubuntu is the most popular distro it only makes sense
.flatpak - flatpak basically works everywhere.

**Comment by KaKi87** [comment]
Hello,
Any news on DEB or Flatpak ?
Thanks

**Comment by 32bitx64bit** [comment]
> Hello, Any news on DEB or Flatpak ? Thanks

Non currently it seems.
I'd just recommend compiling it. As the instructions  are pretty clear.

**Comment by KaKi87** [comment]
Well, I'm actually asking in order to have automated updates, compiling defeats that purpose :sweat_smile:

**Comment by KaKi87** [comment]
@ouwou :pray:
